<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaresSrcImportDialog</class>
 <widget class="QDialog" name="WaresSrcImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
        <item>
         <widget class="QRadioButton" name="PackageRadioButton">
          <property name="text">
           <string>Package:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">SourceButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PackagePathLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PackagePathButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="WareshubWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,0,1,0,0">
          <item row="1" column="2">
           <widget class="QLineEdit" name="UsernameLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="UsernameLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="HubConnectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4">
           <widget class="QLineEdit" name="HubUrlLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="HubRadioButton">
            <property name="text">
             <string>Hub URL:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">SourceButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="PasswordLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="HubLoginButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Log in</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" colspan="2">
           <widget class="QLineEdit" name="PasswordLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="InformationWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Information</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
           <property name="spacing">
            <number>12</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Packager(s):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Description:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="PackagersLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="PackageDescriptionLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QGroupBox" name="WaresGroupBox">
       <property name="title">
        <string notr="true">Available wares (in package) | (on wareshub site)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTabWidget" name="WaresTabWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Simulators</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QListWidget" name="SimListWidget"/>
            </item>
            <item>
             <widget class="openfluid::ui::waresdev::WaresImportFilterWidget" name="SimFilterWidget" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Observers</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QListWidget" name="ObsListWidget"/>
            </item>
            <item>
             <widget class="openfluid::ui::waresdev::WaresImportFilterWidget" name="ObsFilterWidget" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Builder-extensions</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QListWidget" name="ExtListWidget"/>
            </item>
            <item>
             <widget class="openfluid::ui::waresdev::WaresImportFilterWidget" name="ExtFilterWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="SelectionCountLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckoutCurrentVersionCheckBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Try to checkout branch for current OpenFLUID version</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_13"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>openfluid::ui::waresdev::WaresImportFilterWidget</class>
   <extends>QWidget</extends>
   <header location="global">openfluid/ui/waresdev/WaresImportFilterWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>PackageRadioButton</tabstop>
  <tabstop>PackagePathButton</tabstop>
  <tabstop>HubRadioButton</tabstop>
  <tabstop>HubUrlLineEdit</tabstop>
  <tabstop>UsernameLineEdit</tabstop>
  <tabstop>PasswordLineEdit</tabstop>
  <tabstop>HubConnectButton</tabstop>
  <tabstop>WaresTabWidget</tabstop>
  <tabstop>SimListWidget</tabstop>
  <tabstop>ExtListWidget</tabstop>
  <tabstop>ObsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>PackageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>PackagePathButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PackageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>PackagePathLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HubRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>InformationWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="SourceButtonGroup"/>
 </buttongroups>
</ui>
